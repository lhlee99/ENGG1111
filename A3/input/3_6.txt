5
red 2 0 0
blue -1 0 1
green 1 0 1
yellow 0.2 0 0
black 0.2 0 0
