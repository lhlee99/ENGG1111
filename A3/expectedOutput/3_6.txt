red: green
blue: green
green: red blue
yellow: black
black: yellow
